src/foam/dao/FileJournal.js

```diff
@@ -111,6 +112,11 @@ foam.CLASS({
       class: 'Boolean',
       name: 'multiLine'
     },
+    {
+      class: 'Boolean',
+      name: 'multiLinePut',
```

Comment:
I would rather name this "outputMultiline" we don't need to say its specific to put or not, that's an implementation detail of FileJournal.

Reply:
I also wouldn't default it to true, only a select few journals should output this way (scripts, tests, cronjobs) the rest should stick to single-line.